lib/features/composer/presentation/composer_controller.dart

```diff
@@ -173,6 +180,11 @@ class ComposerController extends BaseController {
           _uploadAttachmentsSuccess(success);
         } else if (success is GetEmailContentSuccess) {
           _getEmailContentSuccess(success);
+        } if (success is GetAllIdentitiesSuccess) {
```

Comment:
How about if we can not `get all identities success`? I propose we can use user as we do before.

Reply:
It still works the way you want it to.